### DISCUSSION TITLE
Learnings from two Site Migrations

### DISCUSSION THREAD

**Discussion by jewelsie** [WordPress/developer-blog-content#209] [Topic Ideas]
Hi there! I'm Jasmine, a digital project manager from Ohio. 
I've worked in software for almost a decade but do NOT have a technical background. I just completed the migration of two of my company websites from a headless CMS to WordPress and WOW did I learn a lot. 

I was wondering if this is a topic anyone would be interested in. I could talk to
-Process best practices
-Testing best practices
-Plugins I found useful 
-Resources that were valuable
-The benefit WP is giving us over our headless CMS

**Comment by bph** [comment]
Discussion moved to the Issue after approval. 
Please continue conversation there. 
- #212
